packages/engine/lib/error/src/result_ext.rs

```diff
@@ -112,18 +112,22 @@ pub trait ResultExt {
     where
         C: Context,
         F: FnOnce() -> C;
+
+    // TODO: Temporary, remove before releasing
```

Comment:
Can we add a short comment saying why we've added this? Just for git history and such.

Reply:
Added a reasoning for this function: 67dcb0ac8cbe8979a0338ee955fad5955e537be0